### DISCUSSION TITLE
OTNS how to run multiple commands in one line?

### DISCUSSION THREAD

**Discussion by MartinsMihaeljans** [openthread/openthread#9319] [Q&A] [answered]
Hello,

Is it possible to run two add commands in one take? Like this:
add router x 100 y 100 rr 125; add router x 100 y 150 rr 125
I know this syntax with ; does not work but perhaps there is another.

**Comment by MartinsMihaeljans** [comment, marked as answer]
I figured it out. You can write all commands with enters in notepad and then simply copy them in all together.

**Comment by Aleksei379** [comment]
> Hello,
> 
> Is it possible to run two add commands in one take? Like this:
> add router x 100 y 100 rr 125; add router x 100 y 150 rr 125
> I know this syntax with ; does not work but perhaps there is another.

**Reply by MartinsMihaeljans** [reply]
Hi, @Aleksei379 You simply write your topology in notepad like below and then copy paste it in the OT-NS cli.
add router x 400 y 400 rr 125
add router x 300 y 400 rr 125
add router x 500 y 400 rr 125
add router x 400 y 300 rr 125
add router x 400 y 500 rr 125
